packages/grafana-ui/src/components/TabbedContainer/TabbedContainer.tsx

```diff
@@ -27,7 +27,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme2) => {
     tabContent: css`
       padding: ${theme.spacing(2)};
       background-color: ${theme.colors.background.primary};
-      height: 100%;
+      height: calc(100% - ${theme.components.menuTabs.height}px);
```

Comment:
Thanks so much @Elfo404 for this excellent tip!

Reply:
never use calc :( flex can handle this much better